From a8c08b18f3a2e8f2140c531afaf42715fcab68e7 Mon Sep 17 00:00:00 2001
From: Valentin Heidelberger <31535155+va1entin@users.noreply.github.com>
Date: Fri, 28 Sep 2018 00:05:06 +0200
Subject: [PATCH] Remove average_size hypothesis setting from test_bindings.py
 (#480)

---
 tests/test_bindings.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/tests/test_bindings.py b/tests/test_bindings.py
index 22930cc..d7951a2 100644
--- a/tests/test_bindings.py
+++ b/tests/test_bindings.py
@@ -306,7 +306,6 @@ def test_unpad_not_padded():
 
 
 @given(binary(min_size=0,
-              average_size=128,
               max_size=2049),
        integers(min_value=16,
                 max_value=256)
@@ -320,7 +319,6 @@ def test_pad_sizes(msg, bl_sz):
 
 
 @given(binary(min_size=0,
-              average_size=128,
               max_size=2049),
        integers(min_value=16,
                 max_value=256)
-- 
2.23.0

