From 3ba0a34e28c98c5cb3ba136ec33ba0960339da68 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Sat, 18 Nov 2017 13:51:49 -0600
Subject: [PATCH 2/2] Relax timeout on test(s)

---
 tests/test_pwhash.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/test_pwhash.py b/tests/test_pwhash.py
index 9634c85..1736f27 100644
--- a/tests/test_pwhash.py
+++ b/tests/test_pwhash.py
@@ -439,7 +439,7 @@ def test_argon2id_str_and_verify(password, ops, mem):
        integers(min_value=1024 * 1024,
                 max_value=16 * 1024 * 1024)
        )
-@settings(deadline=1500, max_examples=20)
+@settings(deadline=2000, max_examples=20)
 def test_argon2i_str_and_verify_fail(password, ops, mem):
     _psw = password.encode('utf-8')
     pw_hash = nacl.pwhash.argon2i.str(_psw, opslimit=ops, memlimit=mem)
@@ -448,7 +448,7 @@ def test_argon2i_str_and_verify_fail(password, ops, mem):
 
 
 @given(text(alphabet=PASSWD_CHARS, min_size=5, max_size=20))
-@settings(deadline=1500, max_examples=5)
+@settings(deadline=4000, max_examples=5)
 def test_pwhash_str_and_verify(password):
     _psw = password.encode('utf-8')
 
-- 
2.16.1

